NCTRC/frontend/screens/COVIDErrorScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { SafeAreaView, Text, View, Button, Image, TouchableOpacity } from 'react-native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { AuthStackParamList, AppScreens } from '../Index';
+import { styles } from './Styles';
+
+type CovidErrorScreenNavigationProps = StackNavigationProp<AuthStackParamList, AppScreens.CovidError>;
+interface CovidErrorScreenProps {
+    navigation: CovidErrorScreenNavigationProps;
+    route: { params: ErrorParams };
+}
+
+export type ErrorParams = {
+    reason: string;
+};
+
+const CovidErrorScreen: React.FunctionComponent<CovidErrorScreenProps> = (props) => {
+    let { navigation, route } = props;
+    let { params } = route;
+    let { reason } = params;
+    return (
+        <SafeAreaView style={styles.container}>
+            <TouchableOpacity style={styles.backButton} onPress={() => navigation.pop()}>
+                     <Image source={require('./../assets/backbutton.png')} style={{ width: 75, height: 75 }}></Image>
```

Comment:
Looks like you repeat this code a lot. Probably should turn it into its own component